habitat/plan.sh

```diff
@@ -0,0 +1,120 @@
+pkg_name=chef-dk
+pkg_origin=chef
+pkg_maintainer="The Chef Maintainers <humans@chef.io>"
+pkg_description="The Chef Client"
+pkg_version=$(cat ../VERSION)
+pkg_source=nosuchfile.tar.gz
+pkg_filename=${pkg_dirname}.tar.gz
+pkg_license=('Apache-2.0')
+pkg_bin_dirs=(bin)
+pkg_build_deps=(core/make core/gcc core/coreutils core/git)
+# NOTE: core/openssl is set here to the exact version that
+# core/ruby/2.4.2/20170914220737 was built with. In the future ruby should be
+# automatically built when openssl is updated and we can change our dep back to
+# just `core/openssl`
+pkg_deps=(core/glibc core/busybox-static core/ruby core/libxml2 core/libxslt core/libiconv core/xz core/zlib core/bundler core/openssl/1.0.2j/20170513215106 core/cacerts core/libffi)
+pkg_svc_user=root
+
+do_download() {
```

Comment:
you should be able to drop most of this callback, and do something similar to https://github.com/chef/dcob/blob/a10a7f4b7c967f0bbf2268759375399b4f3d4323/habitat/plan.sh#L22-L31